azure-pipelines.yml

```diff
@@ -323,6 +356,35 @@ extends:
       StrykerBreak: 75
     ACTest: ${{ parameters.RunACTests }}
     releaseParameters:
+      variables:
+        - name: EnableAndroidSupport
+          value: false
```

Comment:
Why are we setting the values in the yml files instead of using the variables from the group?